package.json

```diff
@@ -23,32 +23,30 @@
     "@types/react-dom": "16.0.3",
     "@types/react-router-dom": "^4.2.3",
     "@types/recompose": "^0.26.1",
-    "@types/webpack": "3.8.1",
-    "awesome-typescript-loader": "^4.0.0",
-    "clean-webpack-plugin": "^0.1.18",
+    "@types/webpack": "4.4.21",
+    "awesome-typescript-loader": "^5.2.1",
+    "clean-webpack-plugin": "^1.0.0",
     "cross-env": "^5.1.3",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "faker": "^4.1.0",
     "fast-glob": "^2.2.2",
     "fs": "^0.0.1-security",
     "intern": "^4.2.0",
-    "react-docgen-typescript-loader": "^2.0.3",
-    "react-docgen-typescript-webpack-plugin": "^1.1.0",
```

Comment:
wasn't sure if these were necessary, they seem like they were for storybook based on the npm pages.  They could also be used for typedoc but I dont know why we'd need both

Reply:
@jmadson

Reply:
@tomecho - I believe they were for generating the `typedoc` documentation into the `docs` directory (which is checked-in, as it generates markdown files: https://github.com/Rediker-Software/redux-data-service-react/tree/master/docs

After removing these, please confirm that that running `yarn typedoc` still works as expected (it should generate new markdown files in the docs directory)

Reply:
I just tested `yarn typedoc` locally with this branch and it seems to work fine, removing these dependencies is a good idea 👍